### ISSUE TITLE
Scheduler controls?

### ISSUE THREAD

**Issue by cshamis** [ngageoint/scale-ui#124]
Maybe I'm missing it?  But I can't find how to pause the scheduler in 7.0-rc2.

**Comment by ericsvendsen-mil** [comment]
* Move user information to bottom left (in the footer)
* Change scheduler status dropdown to handle system status. This will be a list of status items (including scheduler) that will have icons which immediately indicate whether something is in trouble. At the bottom of the dropdown there will be a link to a view with more detail for each status item
* Change existing user icon to a power button, which will handle pausing the scheduler, etc.

**Comment by ericsvendsen-mil** [comment]
Waiting on API changes to provide status updates

**Comment by ericsvendsen-mil** [comment]
Create dropdown for power button and add scheduler status back in for now. Merge this issue in and then create a new issue for the status updates.
